### PR TITLE
quick stab at contributing doc

### PR DESCRIPTION
**Pull request by bewest**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+fork. This will probably look something like
+`git push -u origin wip/add-my-widget`.
+
+Now that the commits are available on github, you can click on the
+compare buttons on your fork to create a pull request.  Make sure to
+select [Nightscout's `dev` branch](https://github.com/nightscout/cgm-remote-monitor/tree/dev).
+
+## Comments and issues
+
+We encourage liberal use of the comments, including imgages where
```
Comment (jasoncalabrese) [inline CONTRIBUTING.md]: images

**Comment by jasoncalabrese** [comment]
A nice start just that little typo

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+## Submit a pull request
+
+When you are done working with your prototype, it can be tempting to
+post on popular channels such as Facebook.  We encourage contributors
+to submit their code for review, debate, and release before announcing
+features on social media.
+
+This can be done by checking your code `git commit -avm 'my
+improvements are here'`, the the branch you created back to your own
```
Comment (dduugg) [inline CONTRIBUTING.md]: typo: `the the`

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+select [Nightscout's `dev` branch](https://github.com/nightscout/cgm-remote-monitor/tree/dev).
+
+## Comments and issues
+
+We encourage liberal use of the comments, including imgages where
+appropriate.
+
+## Co-ordination
+
+There is google groups nightscout-core developers list where lots of
```
Comment (dduugg) [inline CONTRIBUTING.md]: "There is a..." might be more readable.

**Comment by dduugg** [comment]
Thanks for doing this!

**Pull request merged**